redi-connect-front/src/components/organisms/RediProgram.tsx

```diff
@@ -21,22 +11,22 @@ const RediProgram = () => {
 
   const programSteps = [
```

Comment:
This all could be moved with help of the i18n return object to the translation.

Reply:
done. components are much cleaner now